src/Components/ClusterRules/ClusterRules.spec.ct.js

```diff
@@ -8,7 +8,7 @@ import _ from 'lodash';
 import getStore from '../../Store';
 import ClusterRules from './ClusterRules';
 import '@patternfly/patternfly/patternfly.scss';
-import data from '../../../cypress/fixtures/ClusterRules/data.json';
+import singleClusterPageReport from '../../../cypress/fixtures/api/insights-results-aggregator/v2/cluster/dcb95bbf-8673-4f3a-a63c-12d4a530aa6f/reports/report-disabled-false.json';
```

Comment:
assuming that you want to have these routes to be similar to API routes as much as possible, why don't you use `'../../../cypress/fixtures/api/insights-results-aggregator/v2/cluster/dcb95bbf-8673-4f3a-a63c-12d4a530aa6f/reports.json'` instead? 

https://github.com/RedHatInsights/insights-results-smart-proxy/blob/master/server/api/v2/openapi.json#L70

Reply:
because that endpoint is parametrized. It is just to make it clear that it is the response when you request it with `?disabled=false`. But if you prefer to use just `reports`, fine with me.